pom.xml

```diff
@@ -14,15 +14,27 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<mule.version>4.4.0</mule.version>
-		<mule.maven.plugin.version>3.8.2</mule.maven.plugin.version>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.source>1.8</maven.compiler.source>
-		<mule.extensions.maven.plugin.version>1.5.0-20221117</mule.extensions.maven.plugin.version>
-		<avio.logger.version>2.0.1</avio.logger.version>
-		<!-- Publish to Maven Central Properties -->
-		<project.github.repository>avioconsulting/mule-health-check-api</project.github.repository>
-		<repository.url>git@github.com:${project.github.repository}.git</repository.url>
-		<nexus.url>https://oss.sonatype.org</nexus.url>
+        <mule.extensions.maven.plugin.version>1.6.0</mule.extensions.maven.plugin.version>
```

Comment:
Please fix the formatting/indentation.  Probably best to make sure the whole file matches format.

Reply:
done